docs/tutorials/regression.ipynb

```diff
@@ -2,204 +2,391 @@
+       "| --- |  --- |   --- | --- |   |            --- |           --- |              --- |    --- |\n",
+       "| i64 |  i64 |   i64 | i64 |   |            i64 |           i64 |              i64 |    i64 |\n",
+       "+===========================================================================================+\n",
+       "|   0 | 2014 |     5 |   2 | … |              0 |          9176 |             2310 | 285000 |\n",
+       "|   1 | 2014 |     5 |   2 | … |              0 |          8595 |             1750 | 285000 |\n",
+       "|   2 | 2014 |     5 |   2 | … |              0 |          9000 |             1850 | 440000 |\n",
+       "|   3 | 2014 |     5 |   2 | … |              0 |          8942 |             1260 | 435000 |\n",
+       "|   4 | 2014 |     5 |   2 | … |              0 |         10836 |             1450 | 430000 |\n",
+       "+-----+------+-------+-----+---+----------------+---------------+------------------+--------+"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
    ],
+   "source": [
+    "# removes columns \"latitude\" and \"longitude\" from table\n",
+    "pricing_columns = pricing.remove_columns([\"latitude\", \"longitude\"])\n",
+    "# removes rows which contain missing values\n",
+    "pricing_values = pricing_columns.remove_rows_with_missing_values()\n",
+    "# removes rows which contain outliers\n",
+    "pricing_outliers = pricing_values.remove_rows_with_outliers()\n",
+    "# For visualisation purposes we only print out the first 5 rows.\n",
+    "pricing_outliers.slice_rows(0,5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "See how to perform further Data-Cleaning at: https://library.safeds.com/en/stable/tutorials/data_processing"
+   ]
+  },
+  {
+   "cell_type": "markdown",
    "metadata": {
     "collapsed": false
-   }
+   },
+   "source": [
+    "## Create Training and Testing Set\n",
+    "\n",
+    "Split the house sales dataset into two tables. A training set, that will be used later to implement a training model to predict the house prices. It contains 60% of the data. The testing set contains the rest of the data. Delete the column `price` from the test set, to be able to predict it later:\n"
+   ]
   },
   {
    "cell_type": "code",
-   "source": [
-    "train_table, testing_table = pricing.split_rows(0.60)\n",
-    "\n",
-    "test_table = testing_table.remove_columns([\"price\"]).shuffle_rows()"
-   ],
+   "execution_count": 18,
    "metadata": {
-    "collapsed": false,
     "ExecuteTime": {
      "end_time": "2024-05-24T11:02:43.020110800Z",
      "start_time": "2024-05-24T11:02:42.978634800Z"
-    }
+    },
+    "collapsed": false
    },
-   "execution_count": 2,
-   "outputs": []
+   "outputs": [],
+   "source": [
+    "train_table, testing_table = pricing_outliers.split_rows(0.60)\n",
+    "\n",
+    "test_table = testing_table.remove_columns([\"price\"]).shuffle_rows()"
```

Comment:
* `remove_columns` is not needed here.
* Why do we need to shuffle the test table?